tests/test_registries.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import markers
+
+from pages.registries import (
+    RegistriesLandingPage,
+    RegistriesDiscoverPage,
+    RegistrationDetailPage
+)
+
+@pytest.fixture
+def landing_page(driver):
+    landing_page = RegistriesLandingPage(driver)
+    landing_page.goto()
+    return landing_page
+
+class TestRegistriesLandingPage:
+    @markers.smoke_test
+    @markers.core_functionality
+    def test_landing_page_loads(self, driver):
```

Comment:
You can leave it if you want, but for the sake of keeping the suite as lean as possible I would remove this. Any other test that includes the landing page will check it loads by default.